<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>370</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>页面置换模拟</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="resultWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>370</width>
      <height>220</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="referLine">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>10</y>
      <width>271</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0 5 2 1 0 7 1 3 5 1 5 4 6 3 4 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>进程顺序：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="fifoButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>FIFO</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>内存大小：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="frameCountSpin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>271</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="clockButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clock</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="lruButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>LRU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>95</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optbutton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OPT</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
